packages/rn-tester-e2e/tests/helpers/utils.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import {driver} from '../../jest.setup';
+
+type PlatformsReference = {
+  ios: string,
+  android: string,
+};
+
+class Utils {
+  async checkElementExistence(locator: string): Promise<boolean> {
+    await driver.$(locator).waitForDisplayed();
+    return driver.$(locator).isDisplayed();
+  }
+
+  async clickElement(locator: string): Promise<void> {
+    await driver.$(locator).waitForDisplayed();
+    await driver.$(locator).click();
+  }
+
+  async getElementText(locator: string): Promise<string> {
+    await driver.$(locator).waitForDisplayed();
+    return driver.$(locator).getText();
+  }
+
+  platformSelect(platforms: PlatformsReference): string {
+    // if something goes wrong, we fallback to ios. But it should never happent, the process will fail way earlier.
+    return platforms[process?.env?.E2E_DEVICE || 'ios'];
+  }
+}
+
+export const iOSLabel = (label: string): string => {
+  return `[label="${label}"]`;
```

Comment:
where can we find the list of all the supported modifiers? We would need the link to direct contributors that wants to write tests. Is it the Appium docs?

Reply:
I'm not sure what you mean with modifiers here, but for the methods to interact with the elements the list is here -> https://webdriver.io/docs/api/element

the `iOSLabel` and `androidWidget` methods are following the pattern from this section (I believe) https://webdriver.io/docs/selectors/#mobile-selectors

cc @szymonrybczak @mateuszm22 @adzironman in case you have better references